QUEST.tsv

```diff
@@ -10,33 +10,33 @@ QUEST_20150317_000009	Please select a starting zone.
 QUEST_20150317_000010	Select Advancement
 QUEST_20150317_000011	Please select a class.
 QUEST_20150317_000012	Bokor Master
-QUEST_20150317_000013	So, you want to learn more about this stone slate. {nl}If you wish, I will show you the will of its creator.
+QUEST_20150317_000013	So you want to learn more about this stone slate. {nl}If you wish, I shall show you the will of its creator.
 QUEST_20150317_000014	Goddess Laima
-QUEST_20150317_000015	Much time will have drifted between us when you see this message. {nl}I am the Goddess of Fate and Wisdom - Laima.
-QUEST_20150317_000016	600 years ago - at the time I am sending this revelation - I saw a great deal of death and grief. {nl}Your seeing of this message likely means that all that I foresaw has come to pass.
-QUEST_20150317_000017	The Demons have started the first cataclysms, and the goddesses have vanished. {nl}I knew all of this in advance and spent great effort to interfere with fate, but my power alone was not enough.
-QUEST_20150317_000018	I have wandered through many far-seeing dreams to find an answer. {nl}You were at the end of them…a Savior.
-QUEST_20150317_000019	However, I knew the Demons would never rest while I kept my power. {nl}I was forced to hide away, until the time of your appearance.
-QUEST_20150317_000020	I resolved to divide my being and subsist inside many of these messages. {nl}I continued to secretly create them, even as the kingdoms flew by over the millennia. 
-QUEST_20150317_000021	Savior. These messages are hidden across axes of time. {nl}Once they are brought together, I will regain my complete form and help against the Demons.
-QUEST_20150317_000022	There may be a weight on your shoulders for this divine task, but it will pass. {nl}Perhaps, in the muddy midst of your trials and tribulations, something will turn around.
+QUEST_20150317_000015	By the time you get this message, much time will have passed. {nl}I am the Goddess of Fate and Wisdom, Laima.
+QUEST_20150317_000016	As I create this revelation, 600 years from your time, I regularly see a great deal of sorrow and death. {nl}If you're seeing this message, then I'm afraid all I've foreseen has likely come to pass.
```

Comment:
Actually, now that I've backtracked it, that line originally read

"When I made this revelation 600 years ago, I saw a lot of grief and death. {Nl} Your seeing this revelation would mean that all I predicted have become reality. {Nl}"

The original context is:  600 years ago, I foresaw a lot of grief and death.  Now because you're seeing this revelation, it means that all I have predicted became reality.

Unfortunately it looks like this line has gone through a bit of "Telephone" and managed to change meaning unintentionally.  The "grief and death" wasn't witnessed by Laima 600 years ago, it was the prediction itself.